src/main/kotlin/app/revanced/patches/music/utils/returnyoutubeusername/ReturnYouTubeUsernamePatch.kt

```diff
@@ -9,8 +9,8 @@ import app.revanced.util.patch.BaseBytecodePatch
 
 @Suppress("unused")
 object ReturnYouTubeUsernamePatch : BaseBytecodePatch(
-    name = "Return YouTube Username",
-    description = "Adds option to replace YouTube Handle with Username in comments using YouTube Data API v3.",
+    name = "Return YouTube Usernames",
```

Comment:
The patch name is still `Return YouTube Usernames`

Just as it should be `Return YouTube Dislike` instead of `Return YouTube Dislikes`, the patch name should be `Return YouTube Username` rather than `Return YouTube Usernames`